enterprise/service/license.go

```diff
@@ -79,7 +79,7 @@ func (s *licenseService) parseClaims(claims jwt.MapClaims) (*enterpriseAPI.Licen
 		return nil, common.Errorf(common.Invalid, err)
 	}
 
-	exp, ok := claims["exp"].(int64)
+	exp, ok := claims["exp"].(float64)
```

Comment:
Fix a bug here, I found I should use float64 to parse the value, otherwise the `ok` will be false and cannot get the right raw value.

Reply:
VerifyExpiresAt is already checked in claims.Valid()

Reply:
I see, but we still need to extract the exp value